src/radio-tile/radio-tile.scss

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+@mixin iui-radio-tile {
+  cursor: pointer;
+  width: $iui-xl * 5;
+  height: 100%;
+  box-sizing: border-box;
+  padding: $iui-s;
+  position: relative;
+  z-index: 1;
+  border: 1px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+  background-color: var(--iui-color-background-1);
+  @media (prefers-reduced-motion: no-preference) {
+    transition: border-color $iui-speed-fast ease-out;
+  }
+
+  // Rounded corners on first tile
+  &:first-of-type {
+    border-radius: $iui-border-radius 0 0 $iui-border-radius;
+  }
+
+  // Rounded corners on last tile
+  &:last-of-type {
+    border-radius: 0 $iui-border-radius $iui-border-radius 0;
+  }
+
+  // All tiles shift 1px left so we don't have 2px thick dividers
+  &:not(:first-of-type) {
+    margin-left: -1px;
+  }
+
+  &-container {
+    display: inline-flex;
+    flex-wrap: wrap;
+    width: fit-content;
```

Comment:
Is this needed? `inline-flex` should handle width according to the content.

Reply:
Removed